lib/cucumber/rails.rb

```diff
@@ -32,7 +32,7 @@
   MultiTest.disable_autorun
 else
   warn "WARNING: Cucumber-rails required outside of env.rb.  The rest of loading is being deferred
-  until env.rb is called. To avoid this warning, move 'gem \'cucumber-rails\', require: false'
+  until env.rb is called. To avoid this warning, move 'gem 'cucumber-rails', require: false'
```

Comment:
Perhaps this became a little off, nesting single quote in single quote.

Reply:
Yep good spot. I'm not able to work on stuff much, but I think we should probably use a squiggly heredoc here.